src/client/src/rt-theme/themes.ts

```diff
@@ -116,6 +116,39 @@ export function createTheme(
       textColor: secondary[2]
     },
 
+    white: colors.spectrum.white.base,
+
+    tile: {
+      backgroundColor: primary.base,
+      textColor: secondary.base,
+      inputColor: secondary['4'],
+
+      primary: {
+        base: accents.accent.base,
+        light: accents.accent['2']
+      },
+
+      bad: {
+        base: accents.bad.base,
+        light: accents.bad['3']
+      },
+
+      good: {
+        base: accents.good.base,
+        light: accents.good['3']
+      },
+
+      accent: {
+        base: accents.accent.base
+      },
+
+      button: {
```

Comment:
This is interesting — this might be another `button` intent/mode/style.

Or, maybe the designers were too optimistic in the palette we'd use in practice and we need to re-think our "secondary" button.

It'd be good to get insight from Carlos on what he thinks of the original styleguide for the button.

Reply:
*The Styleguide Button 😛